applications/api/app/config/config.yml

```diff
@@ -1,16 +1,10 @@
 imports:
-  - { resource: ../../../../global/config/config.yml }
-
-
-#  - { resource: parameters.yml }
-
   - { resource: controllers/_controllers.yml }
   - { resource: commands/commands.yml }
   - { resource: rest/_rest.yml }
-#  - { resource: docs.yml }
   - { resource: logs.yml }
   - { resource: symfony.yml }
 
 
 framework:
-  secret: '%secret%'
+  secret: 'secret'
```

Comment:
Why have you removed the parameter `%` keyword?

This way we would be setting the hardcoded `secret` string as the application string. That could be considered as a security bad practice because it isn't too much complex, and we would be exposing it in our public repo, doesn't we?

Furthermore, this is what make us do not pass [the SymfonyInsight validation](https://insight.symfony.com/projects/8a76d74b-95ea-4a0d-ae52-2d03569b7031/analyses/1)